src/tippy.js

```diff
@@ -11,7 +11,7 @@ const hasBindingChanged = (value, oldValue) => {
 export default (opts = {}) => {
   const init = (el, { value = {}, oldValue = {} }, vnode) => {
     if (!el.getAttribute('title')) {
-      const title = value.title || vnode.data.attrs.title || opts.title
+      const title = value.title || (vnode.data.attrs || {}).title || opts.title
```

Comment:
maybe `vnode.data.attrs && vnode.data.attrs.titles` is better?

Reply:
Sure, updated :)